packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -3217,6 +3328,82 @@ curl -v -X POST \
 ##### Activate Call Factor response example
 
 
+```json
+{
+  "expiresAt": "2015-11-03T10:15:57.000Z",
+  "status": "SUCCESS",
+  "sessionToken": "00Fpzf4en68pCXTsMjcX8JPMctzN2Wiw4LDOBL_9pe",
+  "_embedded": {
+    "user": {
+      "id": "00ub0oNGTSWTBKOLGLNR",
+      "passwordChanged": "2015-09-08T20:14:45.000Z",
+      "profile": {
+        "login": "dade.murphy@example.com",
+        "firstName": "Dade",
+        "lastName": "Murphy",
+        "locale": "en_US",
+        "timeZone": "America/Los_Angeles"
+      }
+    }
+  }
+}
+
+```
+
+#### Activate Email Factor
+
+Activates an `email` factor by verifying the OTP.  The request and response are identical to [activating a TOTP factor](#activate-totp-factor)
+
+##### Activate Email Factor Request Parameters
+
+
+| Parameter    | Description                                         | Param Type | DataType | Required |
+| ------------ | --------------------------------------------------- | ---------- | -------- | -------- |
+| factorId     | `id` of factor returned from enrollment             | URL        | String   | TRUE     |
+| stateToken   | [state token](#state-token) for the current transaction | Body       | String   | TRUE     |
+| passCode     | Passcode received via the email message              | Body       | String   | TRUE     |
+
+
+##### Activate Email Factor Response Parameters
+
+
+[Authentication Transaction Object](#authentication-transaction-model) with the current [state](#transaction-state) for the authentication transaction.
+
+If the passcode is invalid, you receive a `403 Forbidden` status code with the following error:
+
+```http
+HTTP/1.1 403 Forbidden
+Content-Type: application/json
+
+{
+  "errorCode": "E0000068",
+  "errorSummary": "Invalid Passcode/Answer",
+  "errorLink": "E0000068",
+  "errorId": "oaei_IfXcpnTHit_YEKGInpFw",
+  "errorCauses": [
+    {
+      "errorSummary": "Your passcode doesn't match our records. Please try again."
+    }
+  ]
+}
+```
+
+##### Activate Email Factor Request Example
+
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-d '{
+  "stateToken": "007ucIX7PATyn94hsHfOLVaXAmOBkKHWnOOLG43bsb",
+  "passCode": "12345"
+}' "https://${yourOktaDomain}/api/v1/authn/factors/emf1o51EADOTFXHHBXBP/lifecycle/activate"
+```
+
+##### Activate Email Factor Response Example
```

Comment:
`Activate email Factor response example`

Reply:
I appreciate your calling all of these out individually.  It made it a lot easier to fix them